gtsfm/two_view_estimator.py

```diff
@@ -195,11 +215,18 @@ def check_for_degeneracy(
         """ """
         insufficient_inliers = two_view_report.num_inliers_est_model < self._min_num_inliers_acceptance
 
+        H_EF_inlier_ratio = two_view_report.num_H_inliers / (two_view_report.num_inliers_est_model + EPSILON)
+        is_planar_or_panoramic = H_EF_inlier_ratio > MAX_H_INLIER_RATIO
+
         # TODO: technically this should almost always be non-zero, just need to move up to earlier
         valid_model = two_view_report.num_inliers_est_model > 0
-
-        if valid_model and insufficient_inliers:
-            logger.info("Insufficient number of inliers.")
+        if valid_model:
+            logger.info("H_EF_inlier_ratio: %.2f", H_EF_inlier_ratio)
+        if (valid_model and is_planar_or_panoramic) or (valid_model and insufficient_inliers):
+            if is_planar_or_panoramic:
+                logger.info("Planar or panoramic; pose from homography currently not supported.")
+            if insufficient_inliers:
+                logger.info("Insufficient number of inliers.")
```

Comment:
I am against logging at info level. We already have so many lines. Why not log them as debug? Our online benchmarks can have info and above in the logs?

Or do we even needs it in the logs? Why not just log the aggregates? We have the full data in the jsons.